src/store/reducers/load-words.js

```diff
@@ -1,17 +1,15 @@
-import * as actionTypes from './types';
-import { getCorrectWords } from '../service/service';
+import { LOAD_GAME } from '../actions/load-words';
 
 const initialState = {
     correctWords: [],
 };
 
 const reducer = (state = initialState, action) => {
     switch (action.type) {
-        case actionTypes.GET_CORRECT_WORDS:
-            const correctWords = getCorrectWords();
+        case LOAD_GAME:
             return {
                 ...state,
-                correctWords,
+                correctWords: action.correctWords,
```

Comment:
code style comment to consider: if you destructure `action` on line 7 here you can simplify and use object shorthand notation